src/middlewares/getOneArticle.js

```diff
@@ -0,0 +1,36 @@
+import models from '../api/models';
```

Comment:
@MCFrank16 you don't need this in your PR

Reply:
I just use it to check if the article slug exists or not